deploy_tools/cli.py

```diff
@@ -285,6 +292,7 @@ def transact(
     contracts_dir,
     compiled_contracts_path,
     contract_address,
+    value: int,
```

Comment:
I guess this should be 

```suggestion
    value: Optional[int],
```

Reply:
:heavy_check_mark: